From 38460dc7dc1b7ca36c4eba557fadcd097c3a8b71 Mon Sep 17 00:00:00 2001
From: gopisikkan <gopimca.iphone@gmail.com>
Date: Fri, 24 Mar 2017 21:59:39 +0800
Subject: [PATCH] fist commit in test1branch

---
 RSGfile.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/RSGfile.txt b/RSGfile.txt
index bd2ae54..30a1273 100644
--- a/RSGfile.txt
+++ b/RSGfile.txt
@@ -1,3 +1,4 @@
+test1 branch changes
 Contents 
 Sports
 MCA
-- 
2.7.1

